usecases/case_usecase.go

```diff
@@ -304,6 +304,15 @@ func (usecase *CaseUseCase) UpdateCase(ctx context.Context, userId string,
 	return updatedCase, nil
 }
 
+func webhookEventCaseUpdatedFrom(updateCaseAttributes models.UpdateCaseAttributes) models.WebhookEventContent {
+	if updateCaseAttributes.Status != "" {
+		return models.WebhookEventCaseStatusUpdated{
+			CaseStatus: updateCaseAttributes.Status,
+		}
+	}
+	return nil
+}
```

Comment:
Experimentation : use an interface to handel Webhook Event content and create a struct for each custom event

Reply:
My first idea was to direclty implement interface on struct like `updateCaseAttributes models.UpdateCaseAttributes` here, but I'm not totally sure it's a good idea...

Another alternative would be to rollback to the monomorphic event data struct + add suggar functions to create each event (to prevent inline def of `event_data`)

Reply:
yes the second option makes sense to me